extensions/amp-bind/0.1/test/integration/test-bind-impl.js

```diff
@@ -750,6 +762,18 @@ describe
           });
         });
 
+        it('should support binding to CSS classes with arrays, including class+', () => {
+          const element = createElement(
+            env,
+            container,
+            "class='a' [class+]=\"['b','c']\""
+          );
+          expect(toArray(element.classList)).to.deep.equal(['a']);
+          return onBindReadyAndSetState(env, bind, {}).then(() => {
+            expect(toArray(element.classList)).to.deep.equal(['a', 'b', 'c']);
+          });
```

Comment:
Can you add a demo where `class+` is toggle-able?

Reply:
Yes, a lot more tests are needed. I'll note this in the PR description.